src/utils/helperfunctions.ts

```diff
@@ -91,13 +91,21 @@ export function NodeIDStringToBuffer(pk: string): Buffer {
 export function costImportTx(tx: UnsignedTx): number {
   let cost: number = calcBytesCost(tx.toBuffer().byteLength)
   const importTx = tx.getTransaction() as ImportTx
-  importTx.getImportInputs().forEach((input: TransferableInput) => {
-    const inCost = input.getCost()
+  importTx.getImportInputs().forEach((input: TransferableInput): void => {
+    const inCost: number = input.getCost()
```

Comment:
This should be the same cost for each input as of right now, and we do not have any plans to change this at the moment. We could change this to just adding the constant for the cost per input and do the multiplication instead of `forEach` here, but I understand this may be a more forward looking design choice.

Reply:
Going to keep the current design since it's forward looking.